dotnet/src/Connectors/Connectors.Memory.Postgres/IPostgresVectorStoreCollectionSqlBuilder.cs

```diff
@@ -124,13 +126,16 @@ internal interface IPostgresVectorStoreCollectionSqlBuilder
     /// </summary>
     /// <param name="schema">The schema of the table.</param>
     /// <param name="tableName">The name of the table.</param>
-    /// <param name="properties">The properties of the table.</param>
+    /// <param name="propertyReader">The property reader.</param>
     /// <param name="vectorProperty">The property which the vectors to compare are stored in.</param>
     /// <param name="vectorValue">The vector to match.</param>
-    /// <param name="filter">The filter conditions for the query.</param>
+    /// <param name="legacyFilter">The filter conditions for the query.</param>
+    /// <param name="newFilter">The filter conditions for the query.</param>
```

Comment:
Renaming parameters is a notable breaking change. Would it make sense to tackle all the breaking changes at once, including removing the old filter and updating the type, to streamline the transition?

Reply:
This is something we discussed quite extensively recently... The decision was made to obsolete but preserve the current search mechanism, to prevent breakage for current users. We'd remove the legacy search API before releasing, and rename NewFilter to Filter at that point (this creates more churn for users keeping up to date and using the new API).